conftest.py

```diff
@@ -53,3 +55,17 @@ def user() -> dict:
         "zipcode": "12345",
         "city": "example-city"
     }
+
+
+"""Global helper functions."""
```

Comment:
Typically we really use a comment here, so the Python interpreter can ignore that. The only difference would be a doc-string at the beginning of a file.

```suggestion
# Global helper functions.
```